tests/testsuite/build_script.rs

```diff
@@ -4706,18 +4732,24 @@ fn rerun_if_directory() {
     };
 
     // Start with a missing directory.
-    dirty();
+    dirty("", true);
     // Because the directory doesn't exist, it will trigger a rebuild every time.
     // https://github.com/rust-lang/cargo/issues/6003
-    dirty();
+    dirty(
+        "[DIRTY] foo v0.1.0 ([..]): the file `somedir` is missing",
```

Comment:
Another question: what to do about changes to directories? Using ``the file `...` ...`` seems misleading.